candidate-standard/abstract_tests/json/ATS_definition.adoc

```diff
@@ -1,4 +1,4 @@
-[[ats_json_definition]]
+// [[ats_json_definition]] won't work because this gets imported multiple times
```

Comment:
We have a LOT of warnings due to this. Since these files are being included in multiple places, the section link ids are effectively duplicated. We should either remove them or add unique links up in the place where the include gets called. My preference is to remove because they don't get used anywhere and the auto-generated ones are fine.